plugins/services/src/js/reducers/serviceForm/Containers.js

```diff
@@ -539,6 +539,9 @@ module.exports = {
       newState[index] = Object.assign({}, newState[index], {
         image: { id: value, kind: "DOCKER" }
       });
+      if (value === "") {
```

Comment:
Problem is that we operate on the last state, so we need to remove the image attribute if it is not set, so I fear we are not able to remove the condition. Except, we always call the delete before we do the object assign.